### PR TITLE
[13.0] [FIX] l10n_ar: Set trasnlatable the name of the sequence for the new documents.

### PR DESCRIPTION
**Pull request by nicomacr**

### CODE REVIEW

File: addons/l10n_ar/models/l10n_latam_document_type.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def _get_document_sequence_vals(self, journal):
         values.update({'padding': 8, 'implementation': 'no_gap', 'prefix': "%05i-" % (journal.l10n_ar_afip_pos_number),
                        'l10n_latam_journal_id': journal.id})
         if journal.l10n_ar_share_sequences:
-            values.update({'name': '%s - Letter %s Documents' % (journal.name, self.l10n_ar_letter),
+            values.update({'name': _('%s - Letter %s Documents' % (journal.name, self.l10n_ar_letter)),
```
Comment (zaoral) [inline addons/l10n_ar/models/l10n_latam_document_type.py:44]: HI  @nicomacr 
Thank you for the PR :)

Here can you improve the way that the translation is made?
I mean remove the variables inside the _().
Maybe something similar like this?

Also please if you can add the translation term will be great, for localization modules we do it manually updating the pot and po file. 

```suggestion
            values.update({'name': _('%s - Letter %s Documents') % (journal.name, self.l10n_ar_letter),
```

Comment (nicomacr) [reply addons/l10n_ar/models/l10n_latam_document_type.py]: @zaoral Thanks for suggestions. I've already apply this in the current PR.

**Review by bruno-zanotti — APPROVED** [review]
LGTM

**Review by zaoral — APPROVED** [review]

**Review by william-andre — COMMENTED** [review]
Please remove the `[13.0]` in the commit title

`[FIX] l10n_ar: translatable name for the sequence of new documents`

**Comment by nicomacr** [comment]
@william-andre Done!

**Comment by william-andre** [comment]
Just copy paste what I put, you have a typo and the title is too long for git.

**Comment by nicomacr** [comment]
@william-andre now it's ready the commit title!!

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**